code/src/files/index.scss

```diff
@@ -0,0 +1,250 @@
+@import url('https://fonts.googleapis.com/css2?family=Inconsolata:wght@500;600&display=swap');
+
+body {
+  margin: 0;
+  font-family: 'Inconsolata', -apple-system, monospace, 'Segoe UI', 'Roboto',
+    'Oxygen', 'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
+    sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
+* {
+  font-family: 'Inconsolata', monospace;
+  margin: 0;
+  padding: 0;
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+  box-sizing: border-box;
+  -webkit-font-smoothing: antialiased;
+  text-rendering: optimizeLegibility;
+}
+
+// Colors & Other general settings
+$mainRed: #ffadad;
+$lightGray: #f2f0f0;
+$darkGray: #aaaaaa;
+$blk: #000;
+
+@mixin btn {
```

Comment:
You can't imagine how happy I am to see Sass code :) You can be 100% proud of the extra effort you put to learn this pre-processor!